packages/typespec-ts/src/modular/buildSerializeUtils.ts

```diff
@@ -25,7 +25,7 @@ import { ModularCodeModel, Type } from "./modularCodeModel.js";
  */
 export function buildSerializeUtils(model: ModularCodeModel) {
   const serializeUtilFiles = [];
-  for (const serializeType of ["serialize", "deserialize"]) {
+  for (const serializeType of ["deserialize"]) {
```

Comment:
How is serialization being handled?

Reply:
It is now being handled in buildSerializeFunction